tdrs-backend/tdpservice/settings/common.py

```diff
@@ -80,6 +80,7 @@ class Common(Configuration):
         "django_admin_logs",  # logs for admin site
         "corsheaders",
         "django_extensions",
+        "drf_yasg2",
```

Comment:
the documentation here (https://github.com/JoelLefkowitz/drf-yasg) says the original package is being maintained again and that it should be used over this one. any reason for utilizing this version? @jtwillis92

Reply:
Good catch, it does look like they are equivalent at the current time based on the latest release for each but it makes sense to go back to the original package so we can receive future updates.